src/parser/monk/mistweaver/modules/spells/essences/WayOfTheCrane.js

```diff
@@ -31,33 +33,34 @@ class WayOfTheCrane extends Analyzer {
     if (!this.active) {
       return;
     }
+    this._gcd = (1500 / (1 + this.statTracker.hastePercentage(this.statTracker.currentHasteRating)))/1000;
+    this._gcd = Math.max(this._gcd, .75);
```

Comment:
i'm not sure if something to track the gcd exists or not, but this should definitely be a getter because your gcd can change over the course of the fight. for example:

```js
get _gcd() {
    const gcd = 1.5 / (1 + this.statTracker.currentHastePercentage);
    return Math.max(gcd, 0.75);
}
```

Then it will be accurate each time you access it